### PR TITLE
CORE-8415: Filter out unwanted SLF4J bundle in tests

### PR DESCRIPTION
**Pull request by vkolomeyko**
For OSGi tests only bundle from `:testing:slf4jv1` will be used.

### CODE REVIEW

File: buildSrc/src/main/groovy/corda.osgi-test-conventions.gradle [hunk 1]
```diff
@@ -63,9 +63,19 @@ Test-Cases: \${classes;HIERARCHY_INDIRECTLY_ANNOTATED;org.junit.platform.commons
...
+def slf4j2Dependency = libs.slf4j.api.get()
+
+def bundlesSet = files(sourceSets.integrationTest.runtimeClasspath.filter {
+    // Filter out SLF4J bundle as it will be conflicting with special bundle we provide for OSGi testing purposes: slf4jv1
+    !(it.path.contains("${slf4j2Dependency.name}-${slf4j2Dependency.version}"))
+}, configurations.archives.artifacts.files)
+
 def resolve = tasks.register('resolve', Resolve) {
+    // println("bundleSet:")
+    // bundlesSet.forEach { println("    " + it.absolutePath) }
```
Comment (josephzunigadaly) [inline buildSrc/src/main/groovy/corda.osgi-test-conventions.gradle:75]:
```suggestion
```

**Review by josephzunigadaly — APPROVED** [review]
Looks good. Left one suggestion.

**Review by filipesoliveira — APPROVED** [review]
LGTM after the two comments pointed out by Joseph are removed.

**Comment by vkolomeyko** [comment]
Closing without merging as the change is already included into the target branch: ab116fc4656b71fca584b92a3c6cce07526ebb26

**Pull request closed without merging**